source/plugins/go/input/go.mod

```diff
@@ -1,6 +1,8 @@
 module Docker-Provider/source/plugins/go/input
 
-go 1.19
+go 1.21.0
```

Comment:
what's the reason for using 1.21.0 version here?

Reply:
this is automated when doing go mod tidy